docs/admin/openapi.json

```diff
@@ -61,41 +61,6 @@
                 ]
             }
         },
-        "/migrate_system_package": {
```

Comment:
this is api for system_package to system_package2 data migration. Do you want to remove also the code?

Reply:
Code is already gone. This is just removal of forgotten entry in openapi spec.